apis/vshn/v1/vshn_nextcloud.go

```diff
@@ -91,11 +91,12 @@ type VSHNNextcloudServiceSpec struct {
 	// Collabora contains settings to control the Collabora integration.
 	Collabora CollaboraSpec `json:"collabora,omitempty"`
 	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinItems=1
 
-	// FQDN contains the FQDN which will be used for the ingress.
+	// FQDN contains the FQDNs which will be used for the ingress.
```

Comment:
```suggestion
	// FQDN contains the FQDNs array, which will be used for the ingress.
```

`make generate` afterwards